src/Honeybee_IES Luminaire.py

```diff
@@ -816,7 +820,7 @@ def repSpc(originalString):
 
     if _writeRad:  
         if len(_iesFilePath)==1:
-            _iesFilePath = filePath
+            _iesFilePath = _iesFilePath[0]
```

Comment:
Apparently _iesFilePath, = _iesFilePath is faster than _iesFilePath, = _iesFilePath[0]
This is something the I learned recently but it is not really readable. Just sharing.
source: https://twitter.com/raymondh/status/773786851727380480